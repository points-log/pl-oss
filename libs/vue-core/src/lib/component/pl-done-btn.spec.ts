import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLDoneBtn from './pl-done-btn.vue';

Vue.use(Vuetify);

describe('PLDoneBtn', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(PLDoneBtn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
