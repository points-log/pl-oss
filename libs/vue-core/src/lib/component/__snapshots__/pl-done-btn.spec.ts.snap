// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLDoneBtn should match the snapshot 1`] = `
<v-btn-stub
  activeclass=""
  tag="button"
  text="true"
  type="submit"
>
  Done
</v-btn-stub>
`;
